### PR TITLE
aws-lambda-helper does not export a saveRecordToS3() function anymore

### PR DESCRIPTION
**Pull request by hassy**

### CODE REVIEW

**Comment by tomgco** [comment]
Something does not like this PR... I have re-ran and still no luck. (could be the API though)

**Comment by nelsonic** [comment]
https://codeship.com/projects/138820/builds/16565940
codeship-two-tests-failing

**Comment by hassy** [comment]
@nelsonic Same failures as in the classic provider build - unrelated to the changes in the PR, fixing atm.

File: test/parse_sns.test.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ describe('parse_sns', function () {
 describe('get_age', function () {
   it('gets the age in years of a passenger given her birth date', function (done) {
     var DOB = '1986-07-14'; // Jimmy's test!
-    assert(parse_sns.get_age(DOB) === 29);
+    assert(parse_sns.get_age(DOB) === 30);
```
Comment (nelsonic) [inline test/parse_sns.test.js:38]: Happy Birthday @jruts 😜

**Comment by nelsonic** [comment]
Looks good. merging. 👍

**Pull request merged**